lib/Tinify.php

```diff
@@ -72,6 +121,30 @@ function compressionCount() {
     return Tinify::getCompressionCount();
 }
 
+function getRemainingCredits() {
+    return Tinify::getRemainingCredits();
+}
+
+function remainingCredits() {
+    return Tinify::getRemainingCredits();
+}
+
+function getPayingState() {
+    return Tinify::getPayingState();
+}
+
+function payingState() {
+    return Tinify::getPayingState();
+}
+
+function getEmailAddress() {
+    return Tinify::getEmailAddress();
+}
+
+function emailAddress() {
+    return Tinify::getEmailAddress();
+}
+
```

Comment:
These methods are very strange.

They looks very similar. The duplicate should be removed.